nrf_wifi/fw_if/umac_if/src/event.c

```diff
@@ -139,21 +139,25 @@ static void umac_event_connect(struct nrf_wifi_fmac_dev_ctx *fmac_dev_ctx,
 }
 #endif /* CONFIG_NRF700X_STA_MODE */
 
-#ifndef CONFIG_NRF700X_RADIO_TEST
 static enum nrf_wifi_status umac_event_ctrl_process(struct nrf_wifi_fmac_dev_ctx *fmac_dev_ctx,
 						    void *event_data,
 						    unsigned int event_len)
 {
 	enum nrf_wifi_status status = NRF_WIFI_STATUS_SUCCESS;
 	struct nrf_wifi_umac_hdr *umac_hdr = NULL;
+#ifndef CONFIG_NRF700X_RADIO_TEST
```

Comment:
the commit log title and body are same and doesn't really explain the change, please rewrite.